### PR TITLE
adding some basic combat

### PR DESCRIPTION
**Pull request by beeoohbee**

### CODE REVIEW

**Review by GrantDontGknow — APPROVED** [review]

File: main.js [hunk 1]
```diff
@@ -163,16 +173,69 @@ function executeCommand(cmd) {
...
+        playerLocation.agility += getRandomInt(20);
+    }
+}
+
+function display(message) {
+    var outputDiv = document.getElementById("output");
+    outputDiv.innerHTML = outputDiv.innerHTML + '<br>' + message;
+    outputDiv.scrollTop = outputDiv.scrollHeight;
+}
+
```
Comment (Ireallydontknowlandyn) [inline main.js:238]: makes da playder and moonsters attacks dieffer dapendoing on da tyope and da amor.
change the code so different monsters such as master oogway have better health and attack but worse speed.

File: main.js [hunk 2]
```diff
@@ -5,10 +5,17 @@ const map = new Array();
 var playerLocation = {
     floor: 0,
     row: 0,
-    col: 4
-}; // row, colume
+    col: 4,
+    health: 100,
+    attack: getRandomInt(10) + 10,
```
Comment (GrantDontGknow) [inline main.js:10]: start player with more attack but less defense 
attack: getRandomInt(10) +5,
defense: getRandomInt(5) +5,

Comment (Ireallydontknowlandyn) [reply main.js:10]: the player shoul;d be able to choose own stats at the begining of the game and not be stuck with set stats 
/the abillity to add certain stat points based off character choice in beginning 
/have a home page or pop up to choose character in beginning.

File: main.js [hunk 3]
```diff
@@ -57,9 +64,10 @@ function generateMonsters(){
             let r = getRandomInt(10);
             let c = getRandomInt(10);
             var monsterStats = {
+                health: 10 + getRandomInt(50),
                 agility: 1 + floor + getRandomInt((1+floor) * 10),
                 attack: 1 + floor + getRandomInt((1+floor) * 10),
-                defence: 1 + floor + getRandomInt((1+floor) * 10)
+                defense: 1 + floor + getRandomInt((1+floor) * 10)
```
Comment (GrantDontGknow) [inline main.js:70]: Maybe give more defense per floor
3 + floor + getRandomInt((3+floor)  *  10)

**Pull request merged**